frontend/src/components/auth/auth.tsx

```diff
@@ -13,6 +13,10 @@ import {
 } from 'hooks/hooks';
 import { authActions } from 'store/actions';
 import { SignUpForm, SignInForm } from './components/components';
+import { getValidClasses } from 'helpers/helpers';
+import styles from 'components/auth/styles.module.scss';
```

Comment:
```suggestion
import styles from './styles.module.scss';
```
you need to use own styles